src/main/resources/db/migration/V001__Create_notifications_table.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE notifications (
+  id                        BIGSERIAL     PRIMARY KEY,
+  notification_id           VARCHAR(50)   NULL,
+  zip_file_name             VARCHAR(255)  NOT NULL,
+  po_box                    VARCHAR(50)   NULL,
+  service                   VARCHAR(100)  NOT NULL,
+  document_control_number   VARCHAR(100)  NULL,
+  error_code                VARCHAR(25)   NOT NULL,
+  error_description         VARCHAR(255)  NULL,
```

Comment:
just wondering if error code is not null then should error_description be also not null ?

Reply:
we don't have `error_description` field in `error_notifications` table in bulkscan database. so it's better to keep it nullable (at least for now), in case we want to migrate the data.

Reply:
In processor we do store it in `reason` column of events so while migrating we will need to pick it up from this column. We should always have a description else it will be break the contract of notification api.

Reply:
Updated the scriot to make `error_description` field `non null`